docs/eggs/publishing-a-module.md

```diff
@@ -31,6 +33,31 @@ eggs publish myModule --description "A deno module" --checkFormat
 
 For more details, please refer to [configuration](configuration.md#field-information).
 
+Additional options:
+```
+-Y, --yes         Disable confirmation prompts.
+-d, --dry-run     No changes will actually be made, reports the details of what would have been published.
+```
+
+## Programmatically
+
+You just need to import the `publish` function from the latest version of eggs.
+
+```ts
+import { publish } from "https://x.nest.land/eggs@0.3.0/src/commands/publish.ts"
+
+const config = {
+    "description": "Your brief module description",
+    "version": "0.0.1",
+    /* ... */
+};
+
+publish(config, "my-module");
+```
+
+It is the same function as the CLI, features such as logging are still available.
+
+This avoids the need to install eggs. Moreover, every contributor to the project has the same version of eggs.
```

Comment:
Not sure what the "Moreover, every contributor to the project has the same version of eggs." means 😅

Reply:
You can probably put it better than me haha
What I mean is that since with Deno, there is no package.json, the authors can't make sure that all the developers of a module are using the same version of eggs. Therefore, publishing a module in typescript ensures that the version is fixed.

Reply:
Ahhh, this makes sense now. Thanks for the explanation 🙂